generators/client/needle-api/needle-client-webpack.mjs

```diff
@@ -16,17 +16,15 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-const needleClient = require('./needle-client.cjs');
-const constants = require('../../generator-constants.cjs');
+import needleClient from './needle-client.mjs';
+import { CLIENT_WEBPACK_DIR } from '../../generator-constants.mjs';
```

Comment:
We will need to use application variables instead of constants in the future for custom dirs.